--- util/include/platforms/unix/axutil_unix.h	2008-03-10 06:29:37.000000000 +0100
+++ util/include/platforms/unix/axutil_unix.h.oden	2008-05-09 18:17:59.000000000 +0200
@@ -48,15 +48,15 @@ extern "C"
 #define AXIS2_PLATFORM_CHANNEL_PATH        "libhttp_channel.so"
 #define AXIS2_PLATFORM_SSLCHANNEL_PATH     "Unknown"
 
-#define AXIS2_PLATFORM_LOG_PATH            "/usr/local/axis2/log/axutil_log"
-#define AXIS2_PLATFORM_CLIENTLOG_PATH      "/usr/local/axis2/log/axis2_client_log"
-#define AXIS2_PLATFORM_CONFIG_PATH         "/etc/axiscpp.conf"
+#define AXIS2_PLATFORM_LOG_PATH            "/var/log/axis2c/axutil_log"
+#define AXIS2_PLATFORM_CLIENTLOG_PATH      "/var/log/axis2c/axis2_client_log"
+#define AXIS2_PLATFORM_CONFIG_PATH         "/etc/axis2c/axiscpp.conf"
 #define AXIS2_PLATFORM_SECUREINFO          ""
 
     /**
      * Resource that contains the configuration
      */
-#define AXIS2_CONFIGURATION_RESOURCE "/usr/local/axis2/axis2.xml"
+#define AXIS2_CONFIGURATION_RESOURCE "/etc/axis2c/axis2.xml"
 
     /*****************************************************************
      * Library loading and procedure resolution
